entity-framework/core/cli/dotnet.md

```diff
@@ -348,3 +363,4 @@ dotnet ef migrations script 20180904195021_InitialCreate
 
 * [Migrations](xref:core/managing-schemas/migrations/index)
 * [Reverse Engineering](xref:core/managing-schemas/scaffolding)
+* [Compiled models](../performance/advanced-performance-topics.md#compiled-models)
```

Comment:
I know this is just a reference, but should we link to the appropriate doc page for each (relevant) option? That seems more helpful than some generic "additional resources" links

Reply:
Please use xref to page uid instead of referencing md file.

Reply:
We also do that.

The *see also* section is to help guide users who still have questions about a related topic after reaching the end of an article.

Reply:
> Please use xref to page uid instead of referencing md file.